app/views/mentoring/external/show.html.haml

```diff
@@ -5,9 +5,11 @@
       %h1 Share your knowledge with others
       %p
         Mentoring is a fun and rewarding way to reinforce your own learning, while helping others learn and discover things they don’t know.
+        - # TODO
         = link_to "Mentoring also earns reputation.", "#"
       %p.unsure
         Not sure if you can mentor effectively?
+        - # TODO
         = link_to "Some of our best mentors worried too.", "#"
```

Comment:
Not sure what to do here. I want a page of mentor stories, but I don't know what to do in the interim.

Reply:
I think just remove the link part for now and just keep the text. We can add a page with mentor stories later.